actionpack/lib/action_controller/metal/content_security_policy.rb

```diff
@@ -14,6 +14,29 @@ module ContentSecurityPolicy
     end
 
     module ClassMethods
+      # Overrides parts of the globally confiugured Content-Security-Policy
+      # header:
+      #
+      #   class PostsController < ApplicationController
+      #     content_security_policy do |policy|
+      #       policy.base_uri "https://www.example.com"
+      #     end
+      #   end
+      #
+      # Options can be passed similar to +before_action+. For example, pass
+      # +only: :index+ to override the header on the index action only:
```

Comment:
I missed this before.  Needs `<tt>` due to the space:

```suggestion
      # <tt>only: :index</tt> to override the header on the index action only:
```